### PR TITLE
Add test for Event Details page

### PR DESCRIPTION
**Pull request by philcho**
Note there are some commented-out tests there for reference. Sorry for the mess :)

### CODE REVIEW

**Review by BeardOfDan — APPROVED** [review]

**Review by BeardOfDan — APPROVED** [review]

File: src/test/Student.test.js [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
-import Adapter from 'enzyme-adapter-react-16';
 import { shallow } from 'enzyme';
 import { shallowToJson } from 'enzyme-to-json';
 import Student from '../components/Student';
 
-Enzyme.configure({ adapter: new Adapter() });
-
 describe('Student', () => {
   it('should render correctly', () => {
 <<<<<<< HEAD
```
Comment (BeardOfDan) [inline src/test/Student.test.js]: It looks like you have a left over git merge issue

File: src/test/Student.test.js [hunk 2]
```diff
@@ -9,9 +9,22 @@ Enzyme.configure({ adapter: new Adapter() });
...
+      "_id":"5a0d089080bcabc5ac8cb222",
+      "name":"Frodo Baggins",
+      "picture":"https://www.w3schools.com/w3css/img_fjords.jpg",
+      "bio":"I am from someplace and I did something",
+      "__v":0,
+      "comments":[]
+    };
+    const output = shallow(
+      <Student studentData={student} key={student._id} />
+>>>>>>> Update test for Student component to include passed-in props.
```
Comment (BeardOfDan) [inline src/test/Student.test.js]: The arrows and equal signs are signs that you have an unresolved git merge issue

Comment (philcho) [reply src/test/Student.test.js]: Thanks for catching that. It's weird, I couldn't find these git merge artifacts in my local version. Anyhow, I re-pushed the file and the artifacts seem to be gone now.

**Review by BeardOfDan — APPROVED** [review]

File: src/test/Student.test.js [hunk 3]
```diff
@@ -5,11 +5,6 @@ import Student from '../components/Student';
 
 describe('Student', () => {
   it('should render correctly', () => {
-<<<<<<< HEAD
-    const studentData = {};
-    const output = shallow(
-      <Student studentData={studentData} />
-=======
```
Comment (BeardOfDan) [inline src/test/Student.test.js]: Is this what you mean by the new push without the artifacts?

File: src/test/Student.test.js [hunk 4]
```diff
@@ -20,7 +15,6 @@ describe('Student', () => {
     };
     const output = shallow(
       <Student studentData={student} key={student._id} />
->>>>>>> Update test for Student component to include passed-in props.
```
Comment (BeardOfDan) [inline src/test/Student.test.js]: Is this what you mean by the new push without the artifacts?

**Review by BeardOfDan — APPROVED** [review]

**Pull request merged**